lib/rspec/support/ruby_features.rb

```diff
@@ -65,6 +65,12 @@ def supports_exception_cause?
         end
       end
 
+      def ripper_supported?
+        # JRuby supports Ripper but it seems to be unstable
+        # and also reports wrong line number on JRuby 9.0.0.0.
+        defined?(RUBY_ENGINE) && RUBY_ENGINE == 'ruby' && RUBY_VERSION >= '1.9.2'
```

Comment:
Also, I confirmed specs in https://github.com/rspec/rspec-core/pull/2083 properly passed on JRuby 1.7.14 - 1.7.22.